docs/pages/includes/diagnostics/debug-service-intro.mdx

```diff
@@ -0,0 +1,4 @@
+Teleport includes an internal service called "Debug". This service contains
+commands that can help you troubleshoot Teleport issues. The service is used
+through teleport debug commands, which must be executed on the same
+machine/container your Teleport instance runs on.
```

Comment:
> The service is used
through teleport debug commands, which must be executed on the same
machine/container your Teleport instance runs on.

Code block `teleport debug`?

nit: If possible, I would also mention it connects using a unix socket somewhere.

Reply:
I didn't bring it because I felt it would make it more confusing to mention it. For users, using it through `teleport debug` should be transparent; they shouldn't notice any difference if, for example, we swap it to use TCP listeners.